src/Websockettest.js

```diff
@@ -73,12 +78,19 @@ const RMBTTest = (function() {
     /**
      *
      * @param {RMBTTestConfig} rmbtTestConfig
+     * @param {RMBTControlServerCommunication} rmbtControlServer
+     * @param {Logger} newLogger
      * @returns {}
      */
-    function RMBTTest(rmbtTestConfig) {
+    function RMBTTest(rmbtTestConfig, rmbtControlServer, newLogger) {
```

Comment:
The other functions where you use the Logger always initialize by new Logger()
Why the need for an argument here?